lib/installfont.js

```diff
@@ -29,6 +29,12 @@ if(platform.toLowerCase().indexOf('linux') != -1) {
 
 /*======== Module Utilities ========*/
 var isFolder = function (fontPath) {
+
+  fs.exists(fontPath, function (exists) {
```

Comment:
So this works, but the caveat is it will be in a race condition with line 39, because of the nature of async. Want to stick line 38-39 inside this callback? Also you can add yourself as a contributor to the package.json and amend your commit.